DEV_SETUP.md

```diff
@@ -632,7 +632,7 @@ On a clean Ubuntu 18.04 LTS install, the packaged nodejs version is v8. The
 easiest way to get onto the current nodejs v14 is
```

Comment:
```suggestion
easiest way to get onto the current nodejs v16 is
```
you could also change the line above this to reference ubuntu 22.04 LTS, and what appears to be v12 for the included nodejs version (good to confirm).

Reply:
ah, I don't have the logs from my setup now.
I could update that to say
` clean Ubuntu 22.04 LTS install, the packaged nodejs version is expected to be v12`

Reply:
Yeah that seems fine. It doesn't seem too important to specify which version of node is included in the standard ubuntu repositories anyway.

Reply:
Thanks @gherceg 
Made both fixes.